### DISCUSSION TITLE
How should I update the UI after manually deleting photos from the originals folder/location

### DISCUSSION THREAD

**Discussion by jeroendee** [photoprism/photoprism#3250] [Q&A] [answered]
Hi PhotoPrism community, 

Currently I'm testing PhotoPrism (Synology/Portainer setup) and after an initial index I deleted the test assets(photos) from the mounted location. After this I reindexed again. 

The library/originals tab will display "No pictures found" but will show the amount of the previously indexed amount of photos. And the thumbnails are still available. So apparently a reindex does not process the manual deletion of the files (only in the library/originals).

<img width="1776" alt="image" src="https://user-images.githubusercontent.com/220023/223387651-5233cae6-c07c-4ff3-88bc-d75d181058b0.png">

What are the steps now to "correct" the database state?

**Comment by lastzero** [comment, marked as answer]
We have recently added a failsafe check that stops indexing if no files are found in case there is a problem with the volume mount.

**Reply by lastzero** [reply]
Please use the "photoprism reset" command instead.

**Reply by jeroendee** [reply]
That was:
- An amazingly fast response;
- Totally what I was looking for; it solved my problem. 

Also introduced me to the very expansive CLI tool for the first time. Thank you for that.
